integrationtests/tests/fixtures/mod.rs

```diff
@@ -549,21 +549,44 @@ impl FederationTest {
     /// Has every federation node send new consensus proposals then process the outcome.
     pub async fn run_consensus_epochs(&self, epochs: usize) {
         for _ in 0..(epochs) {
-            let consensus = join_all(
-                self.servers
-                    .iter()
-                    .map(|server| Self::consensus_epoch(server.clone(), Duration::from_millis(0))),
-            );
-            if (timeout(Duration::from_secs(15), consensus).await).is_err() {
-                let proposals: Vec<ConsensusProposal> = self
-                    .servers
-                    .iter()
-                    .map(|s| block_on(s.borrow().fedimint.consensus.get_consensus_proposal()))
-                    .collect();
-                panic!("Timed out waiting for consensus, try reducing epochs if proposals are empty: {:?}", proposals);
+            if self
+                .servers
+                .iter()
+                .all(|s| Self::empty_proposal(&s.borrow().fedimint))
```

Comment:
I don't know much about the consensus algorithm. This should only be correct if only all members can delay consensus but a single member can progress it ? (  because of `all`)

Reply:
This just check whether all of the peers have empty epoch proposals which would stall consensus and generally be an error on the test writer's part.